lib/utf8.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+module.exports.decode = function (input) {
+  if (Buffer.isBuffer(input) === false) {
```

Comment:
Instances of `Buffer` are also instances of `Uint8Array` in Node.js 4.x and higher.